src/components/PriceRangePlot/Brush/Brush.tsx

```diff
@@ -83,15 +83,15 @@ export const Handle: React.FC<HandleProps> = ({
         <MinHandle
           height={height}
           x={!isReversed() ? currentPosition - 37 : currentPosition}
-          fill={disabled ? colors.invariant.componentOut3 : colors.invariant.accent1}
+          fill={disabled ? colors.invariant.light : colors.invariant.light}
```

Comment:
these colors shouldn't be the same on enabled and disabled state, check colors on enabled on mockups for adding positions

Reply:
done